azure-mock/upload-documents.ts

```diff
@@ -14,7 +14,9 @@ const uploadFiles = async (
   folderPath: string,
   containerClient: ContainerClient,
 ) => {
-  const files = fs.readdirSync(folderPath);
+  var files = fs.readdirSync(folderPath);
```

Comment:
never `var` since it is very deprecated.
use `let` or `const`. 
Rule of thumb is to use `const` if you can, use `let` if you must and to never ever use `var`.

Reply:
https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994

Reply:
fixed in [58f3ea8](https://github.com/Hochfrequenz/ahbesser/pull/24/commits/58f3ea8c046be1dfcb837f89ed8d3bd634f052ad)